postgresql/manifests/base/upstream/postgres.yaml

```diff
@@ -0,0 +1,326 @@
+---
```

Comment:
Could you please allow generation of multiple manifests by generating them into path per Makefile target?

Reply:
I don't have multiple makefile targets. If you want to add a makefile rule and then adapt the functionality that could be done in a follow up PR

Reply:
Complying with the standards of the repository would demand from you changing literally one line and regenerating manifests. 
Just a note- if you are using `master` reference to generate your postgres, then this change in later PR will break your kustomize files, because directory will change. After that, you will own your manifests directory and only you will be changing them. 

I can't force you to anything. If that's some statement that you are making by refusing code review- then no problem, I can make this change for you.

Reply:
I wont use master for the actual kustomize base. 

Not at all. This PR is already 42 commits long and if I make it perfect it will probably be another 100 long. I think for now I will close this PR until this in in use in energy and then we can reflect on if its right for the rest of the company.